changelog_gen/extractor.py

```diff
@@ -82,13 +82,22 @@ def extract_version_tag(sections: SectionDict) -> str:
     Bugs/Fixes: patch
 
     """
+    version_info_ = BumpVersion.get_version_info("patch")
```

Comment:
explicit tests for this function have been added in a later PR, so currently tested implictly by higher up tests.